File: server-app/src/ctf-server/src/main/java/com/blstream/patronage/ctf/model/Game.java
package com.blstream.patronage.ctf.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Document
public class Game implements BaseModel<String> {

    @Id
    private String id;
    private String name;
    private String description;
    private Date timeStart;
    private Long duration;
    private Integer pointsMax;
    private Integer playersMax;
    private Localization localization;
    private GameStatusType status;
    private List<String> players;
    private String owner;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getTimeStart() {
        return timeStart;
    }

    public void setTimeStart(Date timeStart) {
        this.timeStart = timeStart;
    }

    public Long getDuration() {
        return duration;
    }

    public void setDuration(Long duration) {
        this.duration = duration;
    }

    public Integer getPointsMax() {
        return pointsMax;
    }

    public void setPointsMax(Integer pointsMax) {
        this.pointsMax = pointsMax;
    }

    public Integer getPlayersMax() {
        return playersMax;
    }

    public void setPlayersMax(Integer playersMax) {
        this.playersMax = playersMax;
    }

    public Localization getLocalization() {
        return localization;
    }

    public void setLocalization(Localization localization) {
        this.localization = localization;
    }

    public GameStatusType getStatus() {
        return status;
    }

    public void setStatus(GameStatusType status) {
        this.status = status;
    }

    public List<String> getPlayers() {
        if (players == null)
            players = new ArrayList<String>();
        return players;
    }

    public void setPlayers(List<String> players) {
        this.players = players;
    }

    public String getOwner() {
        return owner;
    }

    public void setOwner(String owner) {
        this.owner = owner;
    }
}

File: windows-phone-client/Ctf/Ctf/ApplicationTools/DataObjects/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;
using Ctf.Communication.DataObjects;

namespace Ctf.ApplicationTools.DataObjects
{

    public class Game : ServerResponse
    {
        public string id { get; set; }
        public string name { get; set; }
        public string description { get; set; }
        public string time_start { get; set; }
        public int duration { get; set; }
        public int points_max { get; set; }
        public int players_max { get; set; }
        public string status { get; set; }
        public Localization localization { get; set; }
        public TeamBase red_team_base { get; set; }
        public TeamBase blue_team_base { get; set; }


        public Game()
        {

        }
        public Game(string name, string description, string id, string time_start, int duration, int points_max, int players_max, Localization localization, TeamBase red_team_base, TeamBase blue_team_base)
        {
            this.id = id;
            this.name = name;
            this.description = description;
            this.time_start = time_start;
            this.duration = duration;
            this.points_max = points_max;
            this.players_max = players_max;
            this.localization = localization;
            this.red_team_base = red_team_base;
            this.blue_team_base = blue_team_base;
        }


        public Game(string name, string description, string time_start, int duration, int points_max, int players_max, Localization localization, TeamBase red_team_base, TeamBase blue_team_base)
        {
            this.name = name;
            this.description = description;
            this.time_start = time_start;
            this.duration = duration;
            this.points_max = points_max;
            this.players_max = players_max;
            this.localization = localization;
            this.red_team_base = red_team_base;
            this.blue_team_base = blue_team_base;
        }
    }

   


}
